hc/accounts/tests/test_profile.py

```diff
@@ -41,10 +52,18 @@ def test_it_adds_team_member(self):
             member_emails.add(member.user.email)
 
         ### Assert the existence of the member emails
-
+        self.assertTrue(member_emails)
+        self.assertTrue(len(member_emails), 2)
```

Comment:
Why **2**?

Reply:
the email of the first member alice who and the new user frank,
``` python
        self.client.login(username="alice@example.org", password="password")

        form = {"invite_team_member": "1", "email": "frank@example.org"}
```